theme-base/_common.scss

```diff
@@ -49,7 +49,7 @@ body {
     }
 
     select,
-    input:not([role="combobox"]):not(.ui-widget) {
+    input:not([role="combobox"]):not(.ui-widget):not(.ui-inputfield) {
```

Comment:
Not sure if this is more efficient for the browser to parse, but can't we write this like
```css
input:not([role="combobox"], .ui-widget, .ui-inputfield) {
```